components/grid/templates/no-data-template.md

```diff
@@ -0,0 +1,74 @@
+---
+title: No Data Template
+page_title: Grid - No Data Template
+description: Use custom no data templates in Grid for Blazor.
+slug: grid-templates-no-data
+tags: telerik,blazor,grid,templates,no,data
+published: True
+position: 38
+---
+
+
+# No Data Template
+
+The `NoDataTemplate` allows you to define custom content when the Grid has no data to show. It overrides the currently available **"No records available."**.
```

Comment:
Consider rewording the second sentence to something like "It lets you change the default "No Records Available" localizable text".